packages/renderer/src/lib/ui/animations.ts

```diff
@@ -0,0 +1,24 @@
+import { cubicOut } from 'svelte/easing';
+
+export function fadeSlide(
```

Comment:
moving this to a separate file will require specific test for the function as well